renkuaqs/plugin.py

```diff
@@ -14,38 +18,36 @@
 
 import os
 import re
+import sys
 import json
 import click
 import rdflib
 from copy import deepcopy
 from pathlib import Path
 
-from renku.core.commands.client import pass_local_client
-from renku.core.commands.graph import Graph
-from renku.core.commands.format.graph import _conjunctive_graph
 from renku.core.models.cwl.annotation import Annotation
+from renku.core.incubation.command import Command
 from renku.core.plugins import hookimpl
-try:
-    from renku.core.models.provenance.provenance_graph import ProvenanceGraph
-    PG_AVAILABLE = True
-except:
-    PG_AVAILABLE = False
+from renku.core.models.provenance.provenance_graph import ProvenanceGraph
+from renku.core.errors import RenkuException
 
 from prettytable import PrettyTable
 from deepdiff import DeepDiff
 
-from .config import AQS_DIR
-#from aqsconverters.models import Run
+from aqsconverters.models import Run
+from aqsconverters.io import AQS_DIR, COMMON_DIR
```

Comment:
Are these necessary? For me, they keep generating errors

Reply:
they will be eventually necessary and can be installed from https://github.com/oda-hub/aqsmodel-converters